recipes/pyraat/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyraat" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyraat-{{ version }}.tar.gz
+  sha256: 4d94274dc5fa7b51047b4e7a48e8c3d5a2640714ba6c42b57ee8e686677dd9eb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - pytest
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - pyraat
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mmcauliffe/Pyraat
+  summary: Interface for running Praat scripts through Python
+  license: MIT
```

Comment:
Upstream has two licenses. Since some here are also maintainer there. Can you please clarify which one is correct?

Reply:
@mmcauliffe is the maintainer for both. My understanding is that MIT is the correct license.

Reply:
@mmcauliffe Can you please confirm this?

In that case the license should be `license: MIT OR other_licencse`